### PR TITLE
changing to C++11 time style

### PR DESCRIPTION
**Pull request by Joilnen**

### CODE REVIEW

File: OgreMain/src/OgreRenderTarget.cpp [hunk 1]
```diff
@@ -487,9 +487,9 @@ namespace Ogre {
     //-----------------------------------------------------------------------
     String RenderTarget::writeContentsToTimestampedFile(const String& filenamePrefix, const String& filenameSuffix)
     {
-        struct tm *pTime;
-        time_t ctTime; time(&ctTime);
-        pTime = localtime( &ctTime );
+        std::time_t t { std::time(nullptr) };
```
Comment (paroj) [inline OgreMain/src/OgreRenderTarget.cpp:490]:
```suggestion
        auto t = std::time(nullptr);
```

File: OgreMain/src/OgreRenderTarget.cpp [hunk 2]
```diff
@@ -487,9 +487,9 @@ namespace Ogre {
...
+        auto pTime { std::localtime(&t) };
```
Comment (paroj) [inline OgreMain/src/OgreRenderTarget.cpp:491]:
```suggestion
        auto pTime = std::localtime(&t);
```

I know its up to taste, but I find this easier to read..

**Comment by paroj** [comment]
the time changes look good now, but it seems your range based for changes ended up here as well, which is not good. (also, there is a white-space related error)

**Comment by Joilnen** [comment]
it passed, after to remove that blank line

**Pull request merged**